dusty/commands/script.py

```diff
@@ -1,5 +1,6 @@
-import textwrap
+# coding=utf-8
```

Comment:
this encodes the python file in unicode, which allows the check mark to be handled properly